scripts/tests/systemd/dns-systemd-resolved.sh

```diff
@@ -15,8 +15,7 @@ sudo cp scripts/tests/systemd/firezone-client.service /etc/systemd/system/
 systemd-analyze security firezone-client
 
 # TODO: Use DNS and not IP
-# HTTPBIN_DNS=172.21.0.100
-HTTPBIN_IP=172.20.0.100
+HTTPBIN_IP=172.21.0.100
```

Comment:
I think I had kept the old httpbin alive because 172.20.0.0 is a CIDR resource, and systemd-resolved control wasn't ready, so this test tries to reach it by IP address.

172.21.0.0 was intentionally not in the resource list because I wanted to show that httpbin was being accessed as a DNS resources.

So I think this is making the systemd-resolved test fail.

Obvious options are, add merge #3643 first, or run two httpbins. Adding 172.21.0.0 as a CIDR resource could work but I prefer not to.

Reply:
Yeah, just trying to keep the number of Docker services lower. What if we just attached the httpbin resource to both `172.21.0.0/24` and `172.20.0.0/24` networks?

Reply:
Can we force Docker's DNS to resolve it to 172.21.0.100 for the gateway then?

Reply:
Good point, doesn't seem so. Reverted back to two services. This PR just contains minor cleanup then.